src/java.base/share/classes/sun/util/cldr/CLDRCalendarDataProviderImpl.java

```diff
@@ -47,8 +45,8 @@
  */
 public class CLDRCalendarDataProviderImpl extends CalendarDataProviderImpl {
 
-    private static Map<String, Integer> firstDay = new ConcurrentHashMap<>();
-    private static Map<String, Integer> minDays = new ConcurrentHashMap<>();
+    private static final Map<String, Integer> firstDay = new ConcurrentHashMap<>();
```

Comment:
We might rename these fields to FIRST_DAY etc.

Reply:
Hm. Usually CAPS naming is used only on immutable constants, but here we have mutable field. (See google style guide for example - https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names).
I like old naming approach.